src/Components/Web/src/Forms/InputNumber.cs

```diff
@@ -40,6 +40,11 @@ private static string GetStepAttributeValue()
     /// </summary>
     [Parameter] public string ParsingErrorMessage { get; set; } = "The {0} field must be a number.";
 
+    /// <summary>
+    /// Gets or sets the format to be used when displaying a number of types: <see cref="float"/>, <see cref="double"/>, <see cref="decimal"/>.
+    /// </summary>
+    [Parameter] public string? Format { get; set; }
```

Comment:
I don't think this works E2E given that https://github.com/dotnet/aspnetcore/pull/60112/files#diff-e63dfe1bc840c170976630c438fc566bcf463606ba3b6c14970d4eac0f898751R61 this is a `type=number` input.